packages/downloads/src/portal-dataset-export-request-parameters.ts

```diff
@@ -0,0 +1,7 @@
+export interface IPortalDatasetExportRequestParameters {
+  host: string;
+  datasetId: string;
+  username: string;
```

Comment:
instead of username & token, accept an [IAuthenticationManager](https://esri.github.io/arcgis-rest-js/api/request/IAuthenticationManager/)

Reply:
that will also allow you to skip sending `host` more info in my general comment